lib/parser/word/entry.rb

```diff
@@ -1,22 +1,41 @@
+require 'digest'
+
 require_relative './entry/dictionary_entry'
 require_relative './entry/inflection'
 
 module Parser
   class Word
     class Entry
       def initialize(doc)
+        @content = doc.inner_html
         @dictionary_entries = doc.xpath('./dict').map { |d| DictionaryEntry.new(d) }
         @inflections = doc.xpath('./infl').map { |i| Inflection.new(i) }
       end
 
-      def to_xml
-        @to_xml ||= %(<entry>#{components.map(&:to_xml).join("\n")}</entry>\n)
+      def bamboo_xml
+        @bamboo_xml ||= <<~XML
+          <oac:hasBody rdf:resource="urn:uuid:idm#{digest}"/>
```

Comment:
This is the second of the three places where this code produces different results. Morphsvc uses a random number here. The code here instead hashes the content of the `<entry>` produced by Morpheus. That way the identifier will be stable but no kind of database is required.

Reply:
that sounds like a good solution